clients/extension/src/background/handlers/requestHandler.ts

```
@@ -46,7 +46,7 @@
 } from '../../utils/tx/getStandardTx'
 import { handleFindAccounts, handleGetAccounts } from './accountsHandler'
 import { handleSendTransaction } from './transactionsHandler'
-
+import { ensureHexPrefix } from '@lib/utils/hex/ensureHexPrefix'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)